samples/core-projects/libraries/new-library/src/Library/project.json

```diff
@@ -1,11 +1,16 @@
 {
-	"dependencies": {
-		"DependencyLibrary":""
+	"version": "1.0.0",
+	"buildOptions": {
+		"debugType": "portable"
 	},
 	"frameworks": {
-		"dotnet55":{
+		"netstandard1.6":{
 			"dependencies": {
-				"System.Runtime":"4.0.0-rc1-*"
+				"NETStandard.Library": "1.6.0",
+				"DependencyLibrary":{
+					"version": "1.0.0",
```

Comment:
I thought we recommended omitting the version element here for project reference targets. Did you mean a specific version?